PowerEditor/installer/nativeLang/corsican.xml

```diff
@@ -1830,8 +1837,10 @@ Impiegà u buttone « ? » à diritta per apre u manuale di l’utilizatore na
 			<fileSaveAsCopySaveButton-tip value="Mantinite u tastu Maius quandu s’appoghja nant’à u buttone Arregistà per apre a copia dopu à l’arregistramentu."/>
 			<autoIndentBasic-tip value="Assicurassi chì l’indentazione di a linea currente (i.e. a linea nova creata da u tastu ENTRÉE) currisponde à l’indentazione di a linea precedente."/>
 			<autoIndentAdvanced-tip value="Attivà l’indentazione astuta per i linguaghji di tipu C è Python. I linguaghji di tipu C includenu :
-C, C++, Java, C#, Objective-C, PHP, JavaScript, JSP, CSS, Perl, Rust, PowerShell è JSON
+C, C++, Java, C#, Objective-C, PHP, JavaScript, JSP, CSS, Perl, Rust, PowerShell è JSON.
 S’è vo selezziunate u modu espertu ma ùn mudificate micca i schedarii in i linguaghji mintulati insù, l’indentazione sterà in modu basicu."/>
+			<!-- Don't translate "&quot;Global override&quot; and &quot;Default Style&quot; -->
+			<global-override-tip value="Permette quì a « Prevalenza glubale » supranerà sti parametri à tutti i stili di tutti i linguaghji di prugrammazione. Preferiscerete di sicuru impiegà piuttostu i parametri « Stilu predefinitu »"/>
```

Comment:
Don't translate "&quot;Global override&quot; and &quot;Default Style&quot;

Reply:
Hello @donho,
 
To be honest I noticed the comment but didn't understand it because the two terms are not used 'as is' elsewhere in the file, excepted for item 21101 where it can be translated.

`			<Item id="2235" name="What is Global override?"/>`
`			<Item id="21101" name="Default style"/>`

Please can you clarify.

Thanks,
Patriccollu.

Reply:
@Patriccollu 
For both Global override & Default style, they are not translatable:

![image](https://github.com/user-attachments/assets/2bfab9ef-7a74-49f8-bb72-6e9f8c5f6b5b)

> `<Folder title="Folder &amp;&amp; Default">`
> `    <Item id="21101" name="Default style"/>`

It's for User Defined Language dialog:

![image](https://github.com/user-attachments/assets/94093502-0d27-4a7b-a763-d64bf6ae45f6)

You should test your translation with Notepad++ binary, instead of just translate it blindly.

Reply:
@donho Thanks for having clarified. I just changed the translated strings to take the comments into account.